test/App.spec.js

```diff
@@ -0,0 +1,46 @@
+import { mount } from '@vue/test-utils';
+import app from '../src/App.vue';
+
+describe('App.vue', () => {
+
+  const wrapper = mount(app);
+  it('renders the todo app', () =>{
+    expect(wrapper.contains('div')).toBe(true);
+  });
+
+  //for some unforsaken reason, this one refuses to work.
+  describe('when the toggleEditModeb function is called', () => {
+    const editButton = wrapper.findAll('button').at(2);
+    editButton.trigger('click');
+    it('opens an input field in a listitem component', () => {
+        expect(wrapper.html()).toContain('input');
+    });
+  });
+
+  describe('when the deleteEntryb function is called', () => {
+    wrapper.vm.deleteEntryb(0);
```

Comment:
Don't access component internal methods or internal state like `wrapper.vm`. Only do that as a last resort, if there is no other way to achieve what you want to do here.

A better way would be to click the delete button (which will hopefully call `deleteEntry` internally).